lib/RTCSession.js

```diff
@@ -390,7 +390,7 @@ module.exports = class RTCSession extends EventEmitter
     debug('init_incoming()');
 
     let expires;
-    const contentType = request.getHeader('Content-Type');
+    const contentType = request.getHeader('Content-Type').toLowerCase();
```

Comment:
Guys, what will happen if the request does not have `Content-Type` header? It will crash.

Reply:
`String(request.getHeader('Content-Type')).toLowerCase()` can fix it.

Reply:
`String(undefined)` returns `"undefined"` string. Not a valid solution. Existence of the header must be checked before getting its value. Can you do a PR for it, please?